router/src/RouterController.ts

```diff
@@ -23,7 +25,7 @@ export class RouterController<T> {
     this.root = root;
   }
 
-  navigateBack(count: number) {
+  navigateBack(route: RouteController<any>, count: number) {
```

Comment:
@sammelanyogi Add the additional parameter at the end

Reply:
Updated.